tools/fix_auth/fix_auth.rb

```diff
@@ -51,21 +52,13 @@ def generate_password
     end
 
     def fix_database_passwords
-      begin
-        ActiveRecord::Base.connection_config
-      rescue ActiveRecord::ConnectionNotEstablished
-        # not configured, lets try again
-        ActiveRecord::Base.logger = Logger.new("#{options[:root]}/log/fix_auth.log")
-        please_establish_connection = true
-      end
+      log_path = "#{options[:root]}/log/fix_auth.log"
       databases.each do |database|
```

Comment:
I think we should drop support for multiple databases.
No one passes multiple databases to be fixed.
That would simplify this logic and allow us to avoid disconnecting from the database.

So the code would simplify to:

```ruby
begin
  ActiveRecord::Base.connection_config
rescue ActiveRecord::ConnectionNotEstablished
  ActiveRecord::Base.logger = Logger.new("#{options[:root]}/log/fix_auth.log")
  ActiveRecord::Base.establish_connection(db_attributes(databases.first))
end
```